storybook/src/foundation/design-tokens/colour.docs.mdx

```diff
@@ -3,78 +3,129 @@
-    <ColorPalette.Tile color={tokens.ams.color["neutral-grey2"]} name="grey" />
-    <ColorPalette.Tile color={tokens.ams.color["neutral-grey3"]} name="dark grey" />
-  </ColorPalette.Section>
+  <ColorPalette.Swatch name="red">
+    <ColorPalette.Tile color="#ec0000" level="60" />
+    <ColorPalette.Tile color="#b70000" level="80" />
+  </ColorPalette.Swatch>
+  <ColorPalette.Swatch name="blue">
+    <ColorPalette.Tile color="#004699" level="60" />
+    <ColorPalette.Tile color="#003677" level="80" />
+  </ColorPalette.Swatch>
+  <ColorPalette.Swatch name="neutral">
+    <ColorPalette.Tile color="#ffffff" level="0" />
+    <ColorPalette.Tile color="#d1d1d1" level="20" />
+    <ColorPalette.Tile color="#767676" level="60" />
+    <ColorPalette.Tile color="#181818" level="100" />
+  </ColorPalette.Swatch>
+  <ColorPalette.Swatch name="lime">
+    <ColorPalette.Tile color="#bed200" level="60" />
+  </ColorPalette.Swatch>
+  <ColorPalette.Swatch name="green">
+    <ColorPalette.Tile color="#00a03c" level="60" />
+  </ColorPalette.Swatch>
+  <ColorPalette.Swatch name="orange">
+    <ColorPalette.Tile color="#ff9100" level="60" />
+  </ColorPalette.Swatch>
+  <ColorPalette.Swatch name="azure">
+    <ColorPalette.Tile color="#009de6" level="60" />
+  </ColorPalette.Swatch>
+  <ColorPalette.Swatch name="yellow">
+    <ColorPalette.Tile color="#ffe600" level="60" />
+  </ColorPalette.Swatch>
+  <ColorPalette.Swatch name="magenta">
+    <ColorPalette.Tile color="#e50082" level="60" />
+  </ColorPalette.Swatch>
+  <ColorPalette.Swatch name="purple">
+    <ColorPalette.Tile color="#a00078" level="60" />
+  </ColorPalette.Swatch>
 </ColorPalette>
 
-## Application
+## Brand tokens
+
+The term 'Brand' represents the highest level in the three-tier token structure of the [NL Design System](https://nldesignsystem.nl/handboek/design-tokens/).
+
+Our brand color tokens have meaningful (semantic) names instead of generic labels like 'blue-60' or 'red-100' that other design systems may use.
+This approach ensures that the correct color is used in the appropriate context.
+Always use a token for its intended purpose based on its name.
+Please reach out to us if you require a different color token.
+
+### Feedback colours
+
+Use these colors for positive and negative feedback, warnings, and messages that need attention.
+Interactive elements in an invalid state use the 'invalid interactive' token, not the 'error feedback' token.
+
+| Token name                   | Color name | Example                                                                                                                    |
+| ---------------------------- | ---------- | -------------------------------------------------------------------------------------------------------------------------- |
+| `ams.color.feedback.error`   | red-60     | <div className="ams-docs-token-example--color" style={{ backgroundColor: 'var(--ams-color-feedback-error-default)' }} />   |
```

Comment:
I would leave the color name out, users wil not be able to search for this color or use it.

Reply:
Or use "Error" here